core/app-layout/Authenticated.tsx

```diff
@@ -21,6 +24,73 @@ function classNames(...classes: any[]) {
   return classes.filter(Boolean).join(" ");
 }
 
+const NavigationButton = () => {
```

Comment:
Should this still be in core?

Reply:
Yes, I would agree that this layout is a bit coupled to the notifications and maybe will be even more to the other features.